.drone.yml

```diff
@@ -103,6 +103,16 @@ steps:
       AWS_SECRET_ACCESS_KEY:
         from_secret: aws_secret
 
+  - name: Build attic-server
+    commands:
+      - nix build '.#nixosConfigurations.attic-server.config.system.build.toplevel'
+      - nix run .#s3uploader $(readlink result)
+    environment:
+      AWS_ACCESS_KEY_ID:
+        from_secret: aws_key
+      AWS_SECRET_ACCESS_KEY:
+        from_secret: aws_secret
+
```

Comment:
Does a drone config even make sense at this point?

Reply:
We only have x86 runners on drone currently.
Yes: we will be able to delete .drone.yml once we have x86 runners and deployed attic in production.
Currently, drone still is responsible to filling our S3 cache.